src/components/SettingsPage.vue

```diff
@@ -26,6 +26,7 @@
                   type="file"
                   class="custom-file-input"
                   id="customFile"
+                  v-on:change="handleProfilePicture"
```

Comment:
use the function to add `profilePicture` 

Refer: https://stackoverflow.com/questions/41803012/v-model-doesnt-support-input-type-file